.orchestra/config/components/ui/mesa.yml

```diff
@@ -2,7 +2,7 @@
 #@ load("@ytt:overlay", "overlay")
 #@ load("/lib/create_component.lib.yml", "single_build_component")
 
-#@ source_url = "https://mesa.freedesktop.org/archive/mesa-20.1.2.tar.xz"
+#@ source_url = "https://mesa.freedesktop.org/archive/mesa-20.1.7.tar.xz"
```

Comment:
@fcremo is working on testing `revng-distributable` on the CI. Once that's in, hopefully we'll be able to bump things more easily.
For now, I'm in favor of this minibump.